src/t_list.c

```diff
@@ -255,22 +254,23 @@ int listTypeNext(listTypeIterator *li, listTypeEntry *entry) {
 }
 
 /* Return entry value at the current position of the iterator. */
```

Comment:
Also mention what it means if the function returns NULL?

Reply:
Done